interfaces/HO-Portal/src/global.scss

```diff
@@ -29,15 +29,30 @@
 // Default styles for a 'styled' select:
 .styled-select {
   padding: 0.45em;
-  border: 1px solid var(--ion-color-medium);
-  border-radius: var(--ion-border-radius);
+  border: 2px solid var(--ion-color-tertiary);
+  background: var(--ion-color-tertiary);
+  color: var(--ion-color-secondary);
+  border-radius: 4px;
+
+  option {
+    color: var(--ion-color-secondary);
+  }
+
+  accent-color: var(--ion-color-secondary);
 }
 
 .horizontal-scroll-container {
   overflow-x: auto;
   -webkit-overflow-scrolling: touch;
 }
 
+.top-header {
+  border-bottom: 1px solid #c2c2c2;
+  &::after {
+    background-image: none;
```

Comment:
Is this to override some Ionic-default styling?

Maybe better to put it on the same selector as the Ionic-styling then.
So `ion-header::after`??